### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by CameronOBrien95**
Please check formatting, naming and logic. @CamA-JCU

### CODE REVIEW

**Comment by CamA-JCU** [comment]
@CameronOBrien95 all of your commit files are empty

**Comment by CameronOBrien95** [comment]
@CamA-JCU sorry about that, I forgot to push the files

File: prac_05/emails.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+email_name = {}
```
Comment (CamA-JCU) [inline prac_05/emails.py:6]: Should you have a docstring at the start of this file?

Comment (CamA-JCU) [inline prac_05/emails.py:6]: Should this list be called emails to follow list naming conventions?

File: prac_05/emails.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+email = input("Email: ")
+valid = False
+while email != "":
+    name = email.split("@")
+    name = name[0]
```
Comment (CamA-JCU) [inline prac_05/emails.py]: Could this be made more efficient by combining into one line using email.split("@")[0]?

File: prac_05/word_occurrences.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+word_count = {}
+sentence = input("Text: ")
+for word in sentence.split(" "):
+    if word in word_count:
```
Comment (CamA-JCU) [inline prac_05/word_occurrences.py:8]: What if you used the Try except method here instead of if else method?

File: prac_05/word_occurrences.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+word_count = {}
```
Comment (CamA-JCU) [inline prac_05/word_occurrences.py:5]: Should there be a docstring at the start of this file?

File: prac_05/hex_colours.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+COLOUR_TO_HEX_CODE = {"DARKORCHID": "#9932cc", "DEEPPINK": "#ff1493", "DODGERBLUE": "1e90ff", "FORESTGREEN": "#228b22",
```
Comment (CamA-JCU) [inline prac_05/hex_colours.py:5]: Should there be a docstring at the start of this file?

**Pull request merged**